### DISCUSSION TITLE
Volume mixer settings

### DISCUSSION THREAD

**Discussion by RensDuijsens** [File-New-Project/EarTrumpet#1638] [Completed ideas]
Make a setting to always show the volume mixer.
So, whenever the PC starts, then show the mixier in a fixed position on the screen.

(I have a 55 inch monitor with lots of room to place all those handy tools)

**Comment by riverar** [comment]
Already exists, try a dev build?  (https://install.eartrumpet.app/dev/EarTrumpet.Package.appinstaller).
